tests/tests_relp.yml

```diff
@@ -111,22 +111,22 @@
       changed_when: false
 
     - name: Check the flows in relp_server0 - 0
-      shell: >-
-        grep '$inputname == "relp_server0"' "{{ __test_relp_server0 }}" |
+      shell: |-
+        grep '$inputname == "relp_server0"' "{{ __test_relp_server0 }}" | \
```

Comment:
it was fine with `>-` and without `\` - unless you ran into some problem?

Reply:
@richm, I'm cleaning up ansible-lint errors next, where `[306] Shells that use pipes should set the pipefail option` is one of them, and it requires to add `set -o pipefail`. Using `|-` makes it easier as follows?
```
      shell: |-
        set -o pipefail
        grep -A 2 "$inputname == \"relp_server0\"" "{{ __test_relp_server0 }}" | \
        grep "call files_output" | wc -l
```

Reply:
yes - using `|-` is recommended for multiline `shell` scripts

Reply:
in fact, we should use `set -euo pipefail` in all of our `shell` to fail fast and catch use of unset variables

Reply:
Ok. I'm opening another pr for ansible-lint. I'm going to add `-eu` and/or `set -euo pipefail` to all the tasks using `shell`. Thanks!